target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -49,6 +49,10 @@ case "$FIRMWARE" in
 		caldata_extract_mmc "0:ART" 0x9000 0x2f20
 		ath10k_patch_mac $(mmc_get_mac_binary ARTMTD 0x12)
 		;;
+	zyxel,wsq50)
```

Comment:
I would suggest doing this as well as other caldata extractions directly via NVMEM, then you can also set the MAC via DTS as well instead of relying on userpace.